### PR TITLE
Update config for running download tests on CI

### PR DESCRIPTION
**Pull request by jrbenny35**
Wanting to see if this is feasible on CI.

### CODE REVIEW

**Comment by dannycoates** [comment]
2 minutes isn't bad 👍

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "test:backend": "nyc --reporter=lcovonly mocha --reporter=min test/backend",
     "test:frontend": "cross-env NODE_ENV=development node test/frontend/runner.js",
     "test:report": "nyc report --reporter=html",
-    "test-integration": "cross-env NODE_ENV=development wdio test/wdio.docker.conf.js",
+    "test-integration": "cross-env NODE_ENV=development ANDROID=1 wdio test/wdio.local.conf.js",
```
Comment (dannycoates) [inline package.json]: Is leaving `ANDROID=1` here intentional? We should be testing the main site here, no?

Comment (jrbenny35) [reply package.json]: Not intentional, I was experimenting. I'll remove this.

**Comment by dannycoates** [comment]
Good stuff @jrbenny35 💯

**Pull request merged**